c7n/commands.py

```diff
@@ -190,7 +191,7 @@ def validate(options):
             ))
         used_policy_names = used_policy_names.union(conf_policy_names)
         if not errors:
-            null_config = Bag(dryrun=True, log_group=None, cache=None, assume_role="na")
+            null_config = set_null_config()
```

Comment:
we use bag for config in lots of places, i'd rather not introduce a new differential path that needs updating in sync with options, the easier fix is just call Bag.empty(**kw)

Reply:
Gotcha. Was also looking at tests/common.py, is there a reason behind having two implementations of Bag? Shouldn't the Config class be moved into utils?

Reply:
other way around, bag should probably be used by tests as well. I think it pre-dates the bag implementation.